### PR TITLE
French Translation : Add new entries and minor modification

### PR DESCRIPTION
**Pull request by ZolOnTheNet**
I compare you last en.yml and add the lack entries. 
I hope a made that correctly.

### CODE REVIEW

**Comment by ZolOnTheNet** [comment]
Rem : when a translate directly the en.json with my traduction to test : the caracteristique is not translate in the Actor sheet
GenesysNoTranslate

**Comment by Assembling-Kings** [comment]
> Rem : when a translate directly the en.json with my traduction to test : the caracteristique is not translate in the Actor sheet, see the red line pencil in the image : GenesysNoTranslate

I tested changing the value and it's working correctly. What might have happened to you is that you didn't reload the world after making the change. This is required for changes on the language files.

File: yaml/lang/fr.yml [hunk 1]
```diff
@@ -0,0 +1,430 @@
+ACTOR:
+  TypeCharacter: Personnage
+  TypeMinion: Sbire
+  TypeRival: Rival
+  TypeNemesis: Nemesis
```
Comment (Assembling-Kings) [inline yaml/lang/fr.yml:5]: These don't seem to be translated.

File: yaml/lang/fr.yml [hunk 2]
```diff
@@ -0,0 +1,430 @@
...
+  TypeVehicle : Vehicule
+
+ITEM:
+  # Character Creation Items
+  TypeAbility: Special Ability
+  TypeArchetype: Archetype
+  TypeCareer: Career
+  TypeInjury: Critical Injury/Hit
```
Comment (Assembling-Kings) [inline yaml/lang/fr.yml:13]: These don't seem to be translated.

File: yaml/lang/fr.yml [hunk 3]
```diff
@@ -0,0 +1,430 @@
...
+    Sections:
+      Armor: Armure
+      Consumable: Consommable
+      Container: Containeur
+      Gear: Equipement
+      Weapon: Armes
+
+  # ActiveEffects System Strings
+  ActiveEffects:
+    Passive: Effects Passifs
```
Comment (Assembling-Kings) [inline yaml/lang/fr.yml:136]: Is it "Effects" or "Effets"?

File: yaml/lang/fr.yml [hunk 4]
```diff
@@ -0,0 +1,430 @@
...
+    Inventory: Inventaire
+    Effects: Effets
+    Journal: Journal
+    Description: Description
+    Data: Données
+    Details: Détails
+    Duration: Durée
+    Miscellaneous: Divers
+    Crew: Equipage
+    Attachments: attaches
```
Comment (Assembling-Kings) [inline yaml/lang/fr.yml:220]: Missing proper capitalization?

File: yaml/lang/fr.yml [hunk 5]
```diff
@@ -0,0 +1,430 @@
...
+    Extreme: Extrême
+    Strategic: Startégique
+
+  # Firing Arc
+  FiringArc:
+    Fore: Avant
+    Aft: Arrière
+    Port: Bâbord
+    Starboard: Tribord
+    All: toutes
```
Comment (Assembling-Kings) [inline yaml/lang/fr.yml:286]: Missing proper capitalization?

File: yaml/lang/fr.yml [hunk 6]
```diff
@@ -0,0 +1,430 @@
...
+      Equipment: Equipement
+      Other: Autre
+    Actor:
+      Adversary: Adversaire
+      Other: Autre
+
+  # Equipment Data
+  Equipment:
+    DamageState:
+      Undamaged: bon état
```
Comment (Assembling-Kings) [inline yaml/lang/fr.yml:302]: Missing proper capitalization?

File: yaml/lang/fr.yml [hunk 7]
```diff
@@ -0,0 +1,430 @@
...
+    Hair: Cheveux
+    Eyes: Yeux
+    NotableFeatures: Caractéristiques notables
+    Notes: Notes
+    XPJournal: Journal des XP
+    Tier: Categorie
+    Activation: Activation
+    Rank: Rang
+    Ranked: à rang
+    Active: actif
```
Comment (Assembling-Kings) [inline yaml/lang/fr.yml:366]: Missing proper capitalization?

File: yaml/lang/fr.yml [hunk 8]
```diff
@@ -0,0 +1,430 @@
...
+    DamageCharacteristic: Dommage lié à la Caractérisitique
+    Critical: Critique
+    Range: Distance
+    Quantity: Quantité
+    EncumbranceThreshold: Seuil de charge
+    Encumbered: Ecombré!
+    RankUp: Rang Supérieur
+    RankDown: Rang Inférieur
+    FreeRankUp: Rang Supérieur (Gratuit)
+    FreeRankDown: Rang Inférieur (gratuit)
```
Comment (Assembling-Kings) [inline yaml/lang/fr.yml:388]: Inconsistent capitalization of "Gratuit".

File: yaml/lang/fr.yml [hunk 9]
```diff
@@ -0,0 +1,430 @@
...
+    SkillNotFound: Compétence non trouvée
+    IsInitiative: Est une Compétence d'initiative?
+    Equipped: Equipé
+    Carried: Porté
+    Dropped: Jetté
+    NotApplicable: N/A
+    EffectiveTier: Catégorie courante
+    Silhouette: Silhouette
+    Speed: vitesse
+    Handling: tenu en main
```
Comment (Assembling-Kings) [inline yaml/lang/fr.yml:414]: Missing proper capitalization?

File: yaml/lang/fr.yml [hunk 10]
```diff
@@ -0,0 +1,430 @@
...
+    HullTrauma: Dommage de la coque
+    HullTraumaThreshold: Seuil du dommage de la coque
+    SystemStrain: Contrainte système
+    SystemStrainThreshold: Seuil de contrainte système
+    Illustration: Illustration
+    Complement: Complément
+    Consumables: Consommable
+    PassengerCapacity: Nombre de passagers
+    EncumbranceCapacity: valeur d'emcombrement
```
Comment (Assembling-Kings) [inline yaml/lang/fr.yml:423]: Missing proper capitalization?

File: yaml/lang/fr.yml (hunk 3 above)
Comment (ZolOnTheNet) [reply yaml/lang/fr.yml:136]: Effets, effect is english ;-)

File: yaml/lang/fr.yml (hunk 1 above)
Comment (ZolOnTheNet) [reply yaml/lang/fr.yml:5]: Rival is the same. A dont put the accent on Nemesis a correct that

File: yaml/lang/fr.yml (hunk 2 above)
Comment (ZolOnTheNet) [reply yaml/lang/fr.yml:13]: Yes, in my tanslate, i have and the yml it's not : i correct that

File: yaml/lang/fr.yml (hunk 4 above)
Comment (ZolOnTheNet) [reply yaml/lang/fr.yml:220]: Yep, just to signal: i dont find in the game, and somme translation can be a  another word... 
I correct all capitalization problem you have signaled ;-), my appologize for that...
I also notice some other errors  that i have correct them...

**Comment by ZolOnTheNet** [comment]
I verify and I hope to no make more mistake, I send you a new pull resquest

**Comment by Cristalnoir** [comment]
Bonjour
Je suis preneur du fichier pour la traduction FR SVP.

**Pull request merged**